src/pages/Options/Market/TradeCard/BiddingPhaseCard/TradeSide.tsx

```diff
@@ -286,4 +269,14 @@ const QuestionMarkStyled = styled(QuestionMark)`
 	height: 8px;
 `;
 
+const MaxButton = styled.button`
+	color: ${(props) => props.theme.colors.buttonHover};
+	background-color: transparent;
```

Comment:
you could use `resetButtonCSS` (from `shared/commonStyles`) which does this reset